src/main/java/com/example/customerdb/controller/PurchaseInfoController.java

```diff
@@ -21,11 +20,28 @@ public List<PurchaseInfo> info() {
         return purchaseInfoService.findAll();
     }
 
-    @GetMapping("/purchase-info/price")
+//    Serviceを使ったpriceのみの一覧表示
+    @GetMapping("/pricelist")
     public List<PurchaseInfoResponse> price() {
         List<PurchaseInfo> purchaseInfos = purchaseInfoService.findAll();
         List<PurchaseInfoResponse> purchaseInfoResponses = purchaseInfos.stream().map(PurchaseInfoResponse::new).toList();
 
         return purchaseInfoResponses;
     }
+
+    @PostMapping("/purchase-info")
+    public void addInfo(@RequestBody PurchaseInfo purchaseInfo) {
+        purchaseInfoService.addInfo(purchaseInfo);
+    }
+
+    @PutMapping("/{id}")
```

Comment:
ここも
purchase-info/1
のようなパス設計にするのが自然ですね。
```suggestion
    @PutMapping("purchase-info/{id}")
```
DELETEも同じく。